package.json

```diff
@@ -30,29 +35,69 @@
   "homepage": "https://github.com/wikibus/heracles#readme",
   "jspm": {
     "registry": "npm",
-    "format": "cjs"
+    "format": "cjs",
+    "directories": {
+      "test": "tests"
+    },
+    "dependencies": {
+      "core-decorators": "^0.13.0",
+      "jsonld": "^0.4.6",
+      "li": "^1.0.1",
+      "linkeddata-vocabs": "^1.0.0",
+      "lodash": "^4.9.0",
+      "rdf-ext": "^0.3.0",
+      "rdf-formats-common": "^0.3.0",
+      "rdf-serializer-jsonld": "^0.3.0",
+      "rxjs": "^5.0.0-rc.1"
+    },
+    "devDependencies": {
+      "core-js": "^2.2.2",
+      "jasmine-core": "^2.4.1",
+      "karma": "^1.3.0",
+      "karma-browserstack-launcher": "^1.0.1",
+      "karma-jasmine": "^1.0.2",
+      "karma-sauce-launcher": "^1.0.0",
+      "karma-systemjs": "^0.15.0",
+      "karma-testingbot-launcher": "^0.1.1",
+      "sinon": "^1.17.3",
+      "systemjs": "^0.19.31",
+      "typescript": "^1.6.2",
+      "whatwg-fetch": "^1.0.0"
+    },
+    "buildConfig": {
+      "transpileES6": true,
+      "minify": true
+    }
+  },
+  "peerDependencies": {
+    "core-js": "^2.4.1"
   },
   "dependencies": {
-    "core-decorators": "^0.12.2",
+    "core-decorators": "^0.13.0",
+    "core-js": "^2.4.1",
     "jsonld": "^0.4.6",
     "li": "^1.0.1",
+    "linkeddata-vocabs": "^1.0.0",
     "lodash": "^4.9.0",
     "rdf-ext": "^0.3.0",
     "rdf-formats-common": "^0.3.0",
-    "rdf-serializer-jsonld": "^0.3.0"
+    "rdf-serializer-jsonld": "^0.3.0",
+    "rxjs": "^5.0.0-rc.1",
```

Comment:
Hm, and actually, `Promise` is included in `core-js` typings. Is that not enough for tsc?